docs/get-started.md

```diff
@@ -1,170 +1,64 @@
-Getting Started
-===============
-
-In order to consume these components you will require a system that
-utilised a bundler (such as [Webpack]) that can process imported assets
-such as images, fonts and [Sass] files. [Create React App] may be able to
-do this but does not provide Server-Side Rendering (SSR). You will also
-need to ensure that you provide an instance of [react-router].
-
-As such, we suggest that you use our specially designed tech stack for
-this purpose. You can start a brand new project or prototype using the
-following steps.
+Prototyping
+===========
 
+This guide explains how to create prototypes using the Home Office Design System.
 
 ### 1. Set up your repository
 
-You should first set up a blank code repository in the provider of your
-choice. For the purposes of this document I will assume [GitHub] as we
-provide extra CI support through [GitHub Actions].
+Set up a blank code repository in [GitHub](https://github.com/) as
+extra CI support is provided through [GitHub Actions](https://github.com/features/actions).
```

Comment:
I expect we can drop this section or move it to the end as an optional extra.

It's really only relevant when a designer wants to collaborate with another designer on the same prototype or when they reach the point of needing to deploy a prototype.

Reply:
Makes sense. I've added this to the new [PR #57](https://github.com/UKHomeOffice/hods-poc/pull/57/files) in the later steps as an option in this file: apps/docs/src/common/pages/get-started/prototype.tsx